kadena-signing-api/src/Kadena/SigningApi.hs

```diff
@@ -96,32 +69,36 @@ instance ToJSON SigningResponse where
 instance FromJSON SigningResponse where
   parseJSON = genericParseJSON compactEncoding
 
+newtype QuickSignRequest = QuickSignRequest
+  { _quickSignRequest_commands :: [CommandSigRequest]
+  } deriving (Eq,Generic)
+
+instance ToJSON QuickSignRequest where
+  toJSON a = object
+    [ "cmds" .= _quickSignRequest_commands a
+    ]
+
+instance FromJSON QuickSignRequest where
+  parseJSON = withObject "QuickSignRequest" $ \o -> do
+    cmd <- o .: "reqs"
+    pure $ QuickSignRequest cmd
+
+newtype QuickSignResponse =
+  QuickSignResponse { unQuickSignResponse :: [ CommandSigRequest ]}
```

Comment:
I just want to double check and make sure the intent here is to use `CommandSigRequest` even though this is `QuickSignResponse`.  If the answer is "yes", perhaps that suggests that `CommandSigRequest` is not quite the right name for that data type.

Reply:
Ah! Good point